idb/postgres/postgres.go

```diff
@@ -289,10 +300,11 @@ func (db *IndexerDb) LoadGenesis(genesis bookkeeping.Genesis) error {
 		if !ok {
 			return fmt.Errorf("LoadGenesis() consensus version %s not found", genesis.Proto)
 		}
+		// is this needed?
```

Comment:
I don't think so, is there a ticket to remove the totals? That should be a good one.

Reply:
yes. there's a ticket for it.